pkg/test/integration_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
Filename could be different. integration_test seems very generic. exporter_collector_test?

Reply:
Makes sense. Thanks